libevmasm/EVMAssemblyStack.cpp

```diff
@@ -78,9 +78,6 @@ std::map<std::string, unsigned> EVMAssemblyStack::sourceIndices() const
 	for (auto const& s: m_sourceList)
 		if (s != CompilerContext::yulUtilityFileName())
 			indices[s] = index++;
-
-	if (indices.find(CompilerContext::yulUtilityFileName()) == indices.end())
-		indices[CompilerContext::yulUtilityFileName()] = index++;
 	return indices;
```

Comment:
The whole function should really be just this:
```c++
return m_sourceList;
```

Reply:
Ah, I did not notice that. Keep the interface as is. But `yulUtilityFileName()` should not be there.